hoomd/pytest/test_box_resize.py

```diff
@@ -0,0 +1,126 @@
+from math import isclose
+import numpy as np
+from pytest import fixture
+
+import hoomd
+
+
+class SimulationSetup:
```

Comment:
This would be easier to read if it was written as one test_ method using `pytest.mark.parameterize` to give it different box1 and box2 inputs. You could also parameterize with a flag to test either `BoxResize` or `BoxResize.linear_variant`.